target/MouseManager.cpp

```diff
@@ -25,9 +25,12 @@
 
 Q_LOGGING_CATEGORY(log_core_mouse, "opf.host.mouse")
 
-MouseManager::MouseManager(QObject *parent) : QObject(parent), mouseMoverThread(new MouseMoverThread()) {
```

Comment:
Probably best not to create a dummy thread instance here if it's going to be created on the fly when needed.